### PR TITLE
feat: ray

### PR DESCRIPTION
**Pull request by martabal**
This PR is a workaround because Ray does not support python 3.11

### CODE REVIEW

**Comment by martabal** [comment]
@hydazz what do you think of this work around ?

File: Dockerfile [hunk 1]
```diff
@@ -198,6 +217,15 @@ RUN \
...
 COPY root/ /
 
+# copy python dependencies
+COPY --from=builder /lsiopy /lsiopy
+
+# copy python 3.10
+COPY --from=builder /usr/bin/python3 /usr/bin/python3
+COPY --from=builder /usr/lib/python3 /usr/lib/python3
+COPY --from=builder /usr/lib/python3.10 /usr/lib/python3.10
+COPY --from=builder /usr/local/lib/python3.10 /usr/local/lib/python3.10
```
Comment (hydazz) [inline Dockerfile:227]: can you minimise the layers, move them to /out/<path> first 😉

Comment (martabal) [reply Dockerfile]: Well the `--copies` argument for `python3 -m venv` prevents to use symlinks, so no need for all these copy 😄

**Comment by martabal** [comment]
Not needed anymore

**Pull request closed without merging**